web/app/js/components/Navigation.jsx

```diff
@@ -260,54 +316,140 @@ class NavigationBase extends React.Component {
   }
 
   render() {
-    const { classes, ChildComponent, ...otherProps } = this.props;
+    const { api, classes, selectedNamespace, ChildComponent, ...otherProps } = this.props;
+    const { namespaces, anchorEl } = this.state;
```

Comment:
should this be `const { namespaces, anchorEl, drawerOpen } = this.state;` for consistency?